po/r-novice-gapminder.ja.po

```diff
@@ -544,30 +544,30 @@ msgid ""
 "  licensor endorses you or your use."
 msgstr ""
 "  your work is derived from work that is Copyright © Software\n"
-"  Carpentry and, where practical, linking to\n"
-"  http://software-carpentry.org/), provide a [link to the\n"
-"  license][cc-by-human], and indicate if changes were made. You may do\n"
-"  so in any reasonable manner, but not in any way that suggests the\n"
-"  licensor endorses you or your use."
+"  Carpentry and, where practical, linking to http://software-carpentry.org/), \n"
+"  [ライセンスへのリンク](https://creativecommons.org/licenses/by/4.0/deed.ja)を\n"
+"  提供し、変更があったらその旨を示さなければなりません。\n"
+"  これらは合理的であればどのような方法で行っても構いませんが、\n"
+"  許諾者があなたやあなたの利用行為を支持していると示唆するような方法は除きます。"
 
 #: r-novice-gapminder/LICENSE.md:34
 msgid ""
 "**No additional restrictions**---You may not apply legal terms or\n"
 "technological measures that legally restrict others from doing\n"
 "anything the license permits.  With the understanding that:"
 msgstr ""
-"**No additional restrictions**---You may not apply legal terms or\n"
-"technological measures that legally restrict others from doing\n"
-"anything the license permits.  With the understanding that:"
+"**追加的な制約は課せません **---あなたは、このライセンスが他の者に\n"
+"許諾することを法的に制限するようないかなる法的規定も技術的手段も\n"
+"適用してはなりません："
 
 #: r-novice-gapminder/LICENSE.md:38
 msgid "Notices:"
-msgstr "Notices:"
+msgstr "ご注意："
```

Comment:
```suggestion
msgstr "注意："
```

Reply:
I think your suggestion is okay but it should match the CC-BY tex [on Creative Commons website](https://creativecommons.org/licenses/by/4.0/deed.ja)

![Screen Shot 2020-10-06 at 12 52 51](https://user-images.githubusercontent.com/5493325/95157096-0c40cf80-07d3-11eb-91bf-deaf3eaf712c.png)

Reply:
OK, let's leave it as-is